wthik.py

```diff
@@ -127,17 +124,22 @@ def travel_schedule(service, calendar_id, num_events=5):
 
 def help_response():
     resp = MessagingResponse()
-    resp.message("""Ask me "Where is Kelley?" to see my current whereabouts or "Travel schedule" to see what's coming up.""")
+    resp.message("""Ask me "Where is Matt?" to see my current whereabouts or "Travel schedule" to see what's coming up.""")
     return str(resp)
 
+@app.route("/test")
+def test():
+    service = _build_service()
+    calendar_id = app.config.get("CALENDAR_ID")
+    return where_is_she(service, calendar_id)
+
```

Comment:
this is removed in master, squashing might get rid of this (was in the branch I had previously shared with you)